core/ecschema-metadata/src/test/Context/SchemaCache.test.ts

```diff
@@ -26,16 +26,20 @@ describe("Schema Cache", () => {
     await cache.addSchema(schema1);
 
     const schema2 = new Schema(context, new SchemaKey("TestSchema"), "ts");
-    await expect(cache.addSchema(schema2)).to.be.rejectedWith(ECObjectsError, "The schema, TestSchema.00.00.00, already exists within this cache.");
+    await expect(cache.addSchema(schema2)).rejects.toThrow(ECObjectsError);
+    await expect(cache.addSchema(schema2)).rejects.toThrow("The schema, TestSchema.00.00.00, already exists within this cache.");
```

Comment:
@hl662 is this the way to do it? This will call addSchema twice just to verify the message of the Error?